lib/live_cluster/client/node.py

```diff
@@ -101,6 +102,56 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class _SysCmd:
+    _uid: int = -1
+
+    @classmethod
+    def set_uid(cls, uid):
+        cls._uid = uid
+
+    def __init__(
+        self,
+        name: str,
+        ignore_error: bool,
+        cmds: list[str],
+        parse_func: Callable[[str], dict[str, Any]],
+    ) -> None:
+        if _SysCmd._uid == -1:
+            raise RuntimeError("set_uid not called")
+
+        self.key = name
+        self.ignore_error = ignore_error
+        self._cmds = cmds
+        self._idx = 0
+        self._parse_func = parse_func
+
+    def parse(self, command_raw_output):
+        result = self._parse_func(command_raw_output)
+        sys_cmd_parser.type_check_basic_values(
+            result
+        )  # Not sure if this is still needed.
+        return result
+
+    def _is_root(self) -> bool:
+        return self._uid == 0
```

Comment:
Should this be checking the class variable?

Reply:
Should not make a difference.

Reply:
If self._uid is ever set it will differ from _SysCmd._uid. Since this is not happening it should work but seems worth noting.